css2/images/anon-block.svg

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   viewBox="0 0 382.66666 258.66666"
+   preserveAspectRatio = "xMidYMid slice"
+   height="202"
+   width="299"
+   xml:space="preserve"
+   id="svg1202"
+   version="1.1"><metadata
+     id="metadata1208"><rdf:RDF><cc:Work
+         rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" /><dc:title></dc:title></cc:Work></rdf:RDF></metadata><defs
+     id="defs1206" /><g
+     transform="matrix(1.3333333,0,0,-1.3333333,0,258.66667)"
+     id="g1210"><g
+       transform="scale(0.1)"
+       id="g1212"><path
+         id="path1214"
+         style="fill:#bfbfbf;fill-opacity:1;fill-rule:evenodd;stroke:none"
+         d="m 24,379.301 h 2832 v 1097.4 H 24 Z" /><path
+         id="path1216"
+         style="fill:#404040;fill-opacity:1;fill-rule:evenodd;stroke:none"
+         d="m 62.0078,432 h 2745.98 V 896 H 62.0078 Z" /><path
+         id="path1218"
+         style="fill:#808080;fill-opacity:1;fill-rule:evenodd;stroke:none"
```

Comment:
Is this Inkscape? A looooot of nonsense :P

Reply:
Yarp. Sadly the only way to convert these from their source files without the image coming out entirely corrupt, from my trials of things that did less craziness.